js/src/settings.js

```diff
@@ -222,16 +222,15 @@
 	
 	//Default settings for the sanitizeHTML function
 	"sanitizeHtmlSettings": {
-		"allowedTags": ['a', 'b', 'br', 'i', 'img', 'p', 'span', 'strong', 'em', 'li', 'ul', 'ol', 'iframe', 'video', 'source'],
-		"allowedAttributes": 
-			{
-				'a': ['href', 'target'],
-				'img': ['src', 'alt'],
-				'p': ['dir'],
-				'video': ['poster', 'controls', 'width', 'height'],
-				'source': ['src'],
-				'iframe': ['src', 'width', 'height', 'frameborder', 'allowfullscreen']
-			}
+	  "allowedTags": ['a', 'b', 'br', 'i', 'img', 'p', 'span', 'strong', 'em', 'li', 'ul', 'ol', 'iframe', 'video', 'source'],
```

Comment:
This is unrelated to the present change, but is it really such a good idea from a security standpoint to allow `source` tags in Mirador by default? It would allow any annotation to execute arbitrary JavaScript in the context of the hosting website, possibly stealing e.g. session tokens.